modules/end-user-guide/partials/proc_adding-projects-to-your-workspace.adoc

```diff
@@ -35,20 +35,7 @@ image::workspaces/che-config-projects-add-git.png[Add Git Project, link="../_ima
 +
 image::workspaces/workspace-devfile-editor.png[Edit Devfile Content, link="../_images/workspaces/workspace-devfile-editor.png"]
 +
-[NOTE]
-.Example: add a project
-====
-To add a project into the workspace, add or edit the following section:
-[source,yaml]
-----
-projects:
-  - name: che
-    source:
-      type: git
-      location: 'https://github.com/eclipse/che.git'
-----
-See the xref:making-a-workspace-portable-using-a-devfile.adoc#devfile-reference_{context}[].
-====
+For demonstration example, see xref:adding-a-git-project-into-a-workspace-using-a-devfile_{context}[Adding a .git project into a workspace using a devfile].
```

Comment:
I'm not sure if it's the most modular way to add xrefs instead of specific examples. Could we maybe put that devfile in examples/ and reuse it in different procedures?

Reply:
What I did there was that I took away two bad examples, create one that is universal, place it in the first procedure, and xrefed it to another, since these two procedures are on the same page and really close to each other. I also wanted to remove the repetition, so I just pointed back + it is not some advanced stuff, so maybe it's not even needed it there, but since it was there originally, I at least used xref. let me know WDYT

Reply:
I think we can just reuse that example. P2 is going to be displaying docs as separate procedures rather than the whole chapter, would be nice to start preparing for it :D

I'll pre-approve though. Everything else looks great.

Reply:
Ok, I will patch it :) Thx for the review. Hope I will downstream this today. Need to have CRW ready up to section 3.6, where developer squat jumps in :)